ch_tools/chadmin/cli/object_storage_group.py

```diff
@@ -129,146 +74,180 @@ def object_storage_group(ctx: Context, config_path: Path, disk_name: str) -> Non
     type=TimeSpanParamType(),
     help=("End of inspecting interval in human-friendly format."),
 )
+@option(
+    "--on-cluster",
+    "on_cluster",
+    is_flag=True,
+    help=("List objects on all hosts in a cluster."),
+)
+@option(
+    "--cluster",
+    "cluster_name",
+    default="{cluster}",
+    help=("Cluster to be cleaned. Default value is macro."),
+)
+@option(
+    "--dry-run",
+    "dry_run",
+    is_flag=True,
+    help=("Do not delete objects. Show only statistics."),
+)
+@option(
+    "--keep-paths",
+    "keep_paths",
+    is_flag=True,
+    help=("Do not delete collected paths of objects from object storage."),
+)
 @pass_context
-def list_objects(
+def clean_command(
     ctx: Context,
-    orphaned: bool,
     object_name_prefix: str,
-    dump_file: Optional[Path],
-    compressed: bool,
-    quiet: bool,
     from_time: Optional[timedelta],
     to_time: timedelta,
-    limit: Optional[int],
+    on_cluster: bool,
+    cluster_name: str,
+    dry_run: bool,
+    keep_paths: bool,
 ) -> None:
     """
-    List S3 objects.
+    Clean orphaned S3 objects.
     """
-    if from_time is not None and from_time <= to_time:
+    if from_time is not None and to_time <= from_time:
         raise click.BadParameter(
-            "from_time parameter must be greater than to_time",
+            "'to_time' parameter must be greater than 'from_time'",
             param_hint="--from-time",
         )
 
     disk_conf: S3DiskConfiguration = ctx.obj["disk_configuration"]
-    now = datetime.now(timezone.utc)
-
-    counter = 0
-
-    with dump_writer(compressed, dump_file) as writer:
-        object_key_to_metadata = collect_metadata(
-            get_disk_metadata_paths(disk_conf.name)
+    listing_table = LISTING_TABLE_PREFIX + disk_conf.name
+    # Create listing table for storing paths from object storage
+    try:
+        execute_query(
+            ctx,
+            f"CREATE TABLE IF NOT EXISTS {listing_table} (obj_path String) ENGINE MergeTree ORDER BY obj_path",
```

Comment:
In what database this table will be created? Suggest to make it configurable through ch-tools configuration file (https://github.com/yandex/ch-tools/blob/master/ch_tools/common/config.py).